phc/api_response.py

```diff
@@ -83,6 +91,35 @@ def get(self, key: str, default=None):
 
         return self.data.get(key, default)
 
+    def get_dataFrame(self, key: str, mapFunc: Callable[[Any], Any] = None):
```

Comment:
Open to suggestions on this name.  Was struggling with what to call it.

Reply:
Maybe `get_as_dataframe`. Do you have an example to show expected usage?

Reply:
```python
>>> from phc import Session
>>> session = Session(account="lifeomic")
>>> from phc.services import Accounts
>>> accounts = Accounts(session)
>>> res = accounts.get_list()
>>> df = res.get_dataFrame("accounts")
>>> df = res.get_dataFrame("accounts", mapFunc=lambda r: r.get("name"))
```

Reply:
@mjtieman updated to:
```python
>>> from phc import Session
>>> session = Session(account="lifeomic")
>>> from phc.services import Accounts
>>> accounts = Accounts(session)
>>> res = accounts.get_list()
>>> df = res.get_as_dataFrame("accounts")
>>> df = res.get_as_dataFrame("accounts", mapFunc=lambda r: r.get("name"))
```

Reply:
Super minor: I would rather not mix camel case and underscores, mind renaming to `get_as_dataframe`?

Reply:
Good suggestion. Updated the code.